tools/contrib/ssp_anomaly_forcing_smooth

```diff
@@ -244,6 +237,17 @@ field_out_wind = ["uas", "vas"]
 
 nfields = len(field_in)
 
+# We can't seem to output in CDF5 format using netCDF4 python interfaces.
+# Instead, write to netCDF-4 and then convert to CDF5.
+output_format = "NETCDF4_CLASSIC"
+def convert_to_cdf5(outfilename):
+    outfilename_cdf5 = outfilename.replace(".nc4", ".nc")
+    subprocess.Popen(
+        f"module load nco && nccopy -k 'cdf5' {outfilename} {outfilename_cdf5} && rm {outfilename}",
+        stdout=subprocess.PIPE,
+        shell="True",
+    )
+
```

Comment:
This also doesn't seem to have happened, as intended.  See /glade/scratch/wwieder/CMIP6-SSP1-2.6